src/pages/actualites/index.page.tsx

```diff
@@ -0,0 +1,86 @@
+import { GetStaticPropsResult } from 'next';
+import React, { useMemo } from 'react';
+
+import { Head } from '~/client/components/head/Head';
+import { Container } from '~/client/components/layouts/Container/Container';
+import { ArticleCard } from '~/client/components/ui/Card/Article/ArticleCard';
+import { LightHero, LightHeroPrimaryText, LightHeroSecondaryText } from '~/client/components/ui/Hero/LightHero';
+import SeeMoreItemList from '~/client/components/ui/SeeMore/SeeMoreItemList';
+import useAnalytics from '~/client/hooks/useAnalytics';
+import { Actualité } from '~/server/actualites/domain/actualite';
+import { isFailure } from '~/server/errors/either';
+import { dependencies } from '~/server/start';
+
+import analytics from './index.analytics';
+import styles from './index.module.scss';
+
+interface ActualitesPageProps {
+	cartesActualites: Array<Actualité>
+}
+
+const MAX_VISIBLE_ACTUALITES_LENGTH = 3;
```

Comment:
Max implique un nombre ?
```suggestion
const MAX_VISIBLE_ACTUALITES = 3;
```